crates/model/src/trade.rs

```diff
@@ -30,7 +33,7 @@ pub struct Trade {
     // Settlement Data
     pub tx_hash: Option<H256>,
     // Fee Policy Data
-    pub fee_policies: Vec<FeePolicy>,
+    pub fee_policies: Vec<(FeePolicy, ExecutedFee)>,
```

Comment:
ditto, I think this should be come one type (with three fields, policy, token, amount)